### DISCUSSION TITLE
createEntityAnywhereFromDBID如何在指定baseapp生成

### DISCUSSION THREAD

**Discussion by Leovany** [kbengine/kbengine#1442] [Q&A]
需求：在弄玩家之间数据的复制（把A玩家数据同步到B）
但因为A和B 可能不在同一个baseapp 上，所以想让AB玩家在同一个baseapp上，方便之后复制处理

1. 若玩家存在，怎么获取玩家实体所在basepp ID ( 判断是否在同一个baseapp 上 )

2. 如何让离线玩家A 在 在线玩家B 所在的baseapp 上生成 ？
   查看文档发现`createEntityAnywhereFromDBID`这个方法，但没看懂baseMB 该如何使用...
   PS : 希望能提供个代码demo参考一下
 
谢谢各位大佬 ~~

![image](https://user-images.githubusercontent.com/2864116/120158668-f632c400-c226-11eb-8c53-fb657872c6f5.png)

**Comment by ybyaoyao** [comment]
BaseApp的序号可以这样 appID = os.getenv("KBE_COMPONENTID") 来获取，先要import os
